CRM/Contact/BAO/Relationship.php

```diff
@@ -1923,18 +1930,20 @@ public static function mergeRelationships($mainId, $otherId, &$sqls) {
   }
 
   /**
-   * Set 'is_valid' field to false for all relationships whose end date is in the past, ie. are expired.
+   * Update 'is_active' field based on the dates of relationship.
    *
    * @return bool
    *   True on success, false if error is encountered.
    * @throws \CiviCRM_API3_Exception
    */
-  public static function disableExpiredRelationships() {
-    $query = "SELECT id FROM civicrm_relationship WHERE is_active = 1 AND end_date < CURDATE()";
-
+  public static function enableDisableAllRelationships() {
```

Comment:
Hmm I understood what "disableExpiredRelationships" meant

"enableDisableAllRelationships" - I'm not so sure...

Reply:
Do you have any suggestion for the name or I may think of something more explanatory

Reply:
updateRelationshipStatusBasedOnDate() ?

Reply:
name changed in function definition and function calls as well